src/Appwrite/Utopia/Database/Validator/Queries/Identities.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Appwrite\Utopia\Database\Validator\Queries;
+
+class Identities extends Base
+{
+    public const ALLOWED_ATTRIBUTES = [
+        'userId',
+        'provider',
+        'providerUid',
+        'providerEmail',
+        'providerAccessTokenExpiry',
```

Comment:
in what scenario do we want to allow querying by the access token expiry ?

Reply:
Let's say we or a developer wants to get all the expired or soon-to-be expired tokens to delete the identity or try and refresh it.

Reply:
okay alright 👍